tests/widget/modal/widget-modal.login-social-redirect.test.js

```diff
@@ -0,0 +1,40 @@
+import { expect, test } from '@playwright/test';
+
+test.only('Modal widget with social callback redirect', async ({ page }) => {
+  await page.goto('widget/modal?journey=LoginWithSocial');
+
+  const loginButton = page.locator('button', { hasText: 'Open Login Modal' });
+  await loginButton.click();
+
+  await page.locator('button', { hasText: 'Continue with Google' }).click();
+
+  await page.waitForNavigation();
+  expect(page.url()).toContain('accounts.google.com', 'redirect_uri');
+});
+
+test.only('Modal widget with social callback redirect no local auth', async ({ page }) => {
+  await page.goto('widget/modal?journey=LoginWithSocialAndLocalAuth');
+
+  const loginButton = page.locator('button', { hasText: 'Open Login Modal' });
+  await loginButton.click();
+
+  await page.waitForNavigation();
+  expect(page.url()).toContain('accounts.google.com', 'redirect_uri');
+});
+
+test.only('Modal widget with social callback local authentication', async ({ page }) => {
```

Comment:
@cerebrl what is your view on multiple test cases in the same file? I grouped them together as they are very similar variations on the same thing - the scenarios around local authentication. I'm happy to split them out into different files if you prefer.